Rakefile

```diff
@@ -1,8 +1,9 @@
-require 'bundler/setup'
```

Comment:
That's what bundler's binstubs are for. In general, I think this decision – how bundler is invoked (via `bundle exec` or binstubs, etc) – should be left in the hands of the user.